itemManagement/templates/itemManagement/item_details_admin.html

```diff
@@ -131,6 +131,7 @@
             </div>
 
             <a href="" class="m-2">view history report</a>
+        </div>
```

Comment:
Is this an accident or was a div not closed before?

Reply:
not closed before